features/util.py

```diff
@@ -84,9 +88,19 @@ def lxc_get_property(name: str, property_name: str, image: bool = False):
         return value
 
 
-def build_debs(
-    series: str, output_deb_dir: str, cache_source: bool
-) -> List[str]:
+def repo_state_hash(include_features: bool = False):
+    git_log_cmd = ["git", "log", "-1"]
```

Comment:
I think we can only use the hash of the latest commit here: `git rev-parse HEAD`

Reply:
Yes good point! Because all we really need is the commit hash.